pkg/Microsoft.ML.Featurizers/Microsoft.ML.Featurizers.nupkgproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Additional ML.NET featurizers for AutoML</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.4.0-preview3*" />
```

Comment:
This should be a `ProjectReference`. See our other .nupkgproj files for an example.

Reply:
Sounds good. Set it back to the `ProjectReference`.

---
In reply to: [340748512](https://github.com/dotnet/machinelearning/pull/4413#discussion_r340748512) [](ancestors = 340748512)